File: c/src/common/pastafazul.c

#include "pastafazul.h"

soup_t * pastafazul_new()
{
	soup_t * soup = soup_new();
	soup->name = "Pasta Fazul";
	arraylist_string_add( soup->ingredients,"1 Pound tomatos");
	arraylist_string_add( soup->ingredients,"1/2 cup pasta");    
	arraylist_string_add( soup->ingredients,"1/2 cup diced carrots");          
	arraylist_string_add( soup->ingredients,"1 cup tomato juice");             
   return soup;
}

File: php/src/common/PastaFazul.php
<?

require_once 'Soup.php';


class PastaFazul extends Soup
{
	function __construct()
	{
      $this->soupName = "Pasta Fazul";
      $this->soupIngredients = [
        "1 Pound tomatos",
        "1/2 cup pasta",
        "1/2 cup diced carrots",
        "1 cup tomato juice",
      ];
    }
}
